arch/arm64/boot/dts/qcom/sm7150-xiaomi-tucana.dts

```diff
@@ -138,6 +225,42 @@
 	enable-gpios = <&tlmm 67 GPIO_ACTIVE_HIGH>;
 };
 
+&panel {
+	compatible = "xiaomi,f4-41-06-0a";
```

Comment:
I thought you were telling me that you found the name of the panel. Maybe you'll use it right away?